package.json

```diff
@@ -6,8 +6,8 @@
   "scripts": {
     "lint:js": "eslint --fix",
     "lint:html": "tidy -config tidy_config.txt",
-    "lint:css": "stylelint",
-    "lint": "npm run lint:js -- --ext .js && npm run lint:html *.html && npm run lint:css **/*.css",
+    "lint:css": "stylelint --fix",
+    "lint": "npm run lint:js *.js public/js/*.js && npm run lint:html *.html && npm run lint:css public/css/*.css",
```

Comment:
Oh okay, so I'm assuming that we just have to make sure that all stylesheets have to be inside the `public` dir right?

Reply:
Why do we need the additional pattern for lint?

Reply:
Yeah, I mean it could be updated later on but I doubt we'll have CSS files outside of `public` unless we end up using a CSS preprocessor with or a FE framework/library.
Also, do you think `index.html` should be moved to `public`, that way it follows a more standard express project structure and it would make it easier to deploy the code?

> Why do we need the additional pattern for lint?

The `--fix` in `stylelint --fix` would enable it to auto-fix what it can (similar to `eslint --fix`).
ESLint defaults to `.js` for `--ext` (according to the manual) so I thought it would be redundant and removed it after seeing that the command wasn't picking up all the `.js` files when I was running it locally.

Reply:
+1 to moving `index.html` inside of `public`